src/main/java/step2_2/view/OutputView.java

```diff
@@ -0,0 +1,48 @@
+package step2_2.view;
+
+import static java.lang.String.format;
+
+import java.util.Map;
+import step2_2.domain.LottoBundle;
+import step2_2.domain.LottoResult;
+import step2_2.domain.LottoReward;
+
+public class OutputView {
+
+    private static final String RESULT_PRINT_MESSAGE = "당첨 통계";
+    private static final String LINE = "------------";
+    private static final String WINNER_PRINT_FORMAT = "%d개 일치 (%s원)- %d개\n";
+    private static final String YIELD_PRINT_FORMAT = "총 수익률은 %.2f입니다\n";
+    private static final int MATCH_COUNT_MIN = 3;
+    private static final int MATCH_COUNT_MAX = 6;
+
+    private OutputView() {
+    }
+
+    public static void printLottoBundle(LottoBundle lottoBundle) {
+        System.out.println(lottoBundle.getLottoCount() + "개를 구입하셨습니다");
+        System.out.println(lottoBundle);
+    }
+
+    public static void printLottoResult(LottoResult lottoResult) {
+        System.out.println(RESULT_PRINT_MESSAGE);
+        System.out.println(LINE);
+        printRewardMap(lottoResult.getRewardMap());
+        printYield(lottoResult);
+    }
+
+    private static void printRewardMap(Map<LottoReward, Long> rewardMap) {
+        for (int matchCount = MATCH_COUNT_MIN; matchCount <= MATCH_COUNT_MAX; matchCount++) {
+            LottoReward lottoReward = LottoReward.fromMatchCount(matchCount);
+            printReward(lottoReward, rewardMap.get(lottoReward));
+        }
```

Comment:
다음과 같이 해당 상수들을 사용하지 않는 방향으로 코드롤 작성해보면 어떨까요?
```suggestion
        for (Map.Entry<LottoReward, Long> lottoReward : rewardMap.entrySet()) {
            printReward(lottoReward.getKey(), rewardMap.get(lottoReward.getKey()));
        }
```

다만 이럴 경우, 번호가 0개 일치하는 경우도 함께 출력되므로
`printReward()`에서 필터링을 한 번 해주면 될 것 같습니다.

처음부터 `rewardMap`에 번호가 0개 일치하는 경우를 담지 않는 것도 하나의 방법인데
이 부분도 한 번 고민해보시면 성장에 도움이 되실 것 같습니다.

Reply:
다른 리뷰에서 `LottoNumber의 상수를 public으로 열어서 함께 활용할 수도 있을 것 같아요.` 
라고 남겨주신 부분에서 상수를 public으로 선언할 수 있는 것은 외부에서 변경될 여지가 없기 때문이라고 생각합니다.
그렇다면 이 부분에서도 `LottoReward` 에 3과 6을 상수로 선언하고 public으로 열어 활용할 수 있다고 생각하는데 그런 구성은 어떨지 리뷰어님의 의견이 궁금합니다.

Reply:
`Lotto`에서 `LottoNumber`의 상수를 활용할 수도 있겠다고 말씀드렸던 이유는
`Lotto`의 `initDefaultLottoNumbers()`에서 `IntStream.range(1, 46)` 을 `IntStream.rangeClosed(1, 45)`로 수정할 수 있고
`1`과 `45`는 `LottoNumber`에 상수로 이미 정의가 되어 있으며 동일한 의미로 쓰일 수 있기 때문이었습니다.

말씀하신 것처럼 `LottoReward`에 `3`, `6`을 상수로 선언하셔서 사용하는 건 괜찮다고 생각하는데
외부에서 `3`과 `6`을 동일한 의미로 사용하지 않는다면 `public`으로 열어두는 건 개인적으로 반대하는 입장입니다.
현재 `step2_2` 패키지에서는 `step2` 패키지처럼 `LottoResultDto`가 없어서 기존 상수를 활용하는 부분이 없어서
상수를 사용하지 않는 방향으로 개선하는 방법을 말씀드렸다고 보시면 될 것 같아요.